install.py

```diff
@@ -204,7 +204,10 @@ def go(ui, args, answerfile_address, answerfile_script):
         results['extra-repos'] += extra_repo_defs
         logger.log("Driver repos: %s" % str(results['extra-repos']))
 
+        mounter = disktools.DeviceMounter()
```

Comment:
It seems a bit wasteful to recreate this and reparse arguments every time this is used. Can you use a singleton (or just a module level variable)? If you change the argument handling so that `--mount=` is passed to the installer, it could be created during argument parsing.

Reply:
Yes, that for sure. I was going to implement a singleton while adding delay to avoid having the delay every time (yes, I saw you suggested to ignore it as we settle the devices... possibly we need to check if we did the settle before the first `DeviceMounter` usage). I'll add the parse in a single place.

Reply:
So... done I suppose!